cli/src/config/templates.toml

```diff
@@ -402,3 +402,16 @@ coalesce(
   "o",
 )
 '''
+
+git_format_patch_email_headers = '''
+  concat(
+    "From " ++ commit_id ++ " Mon Sep 17 00:00:00 2001\n",
```

Comment:
nit: can you add comment about `Mon Sep 17 00:00:00 2001`? It looks like a bug, but isn't.